src/crewai/crew.py

```
@@ -388,12 +402,12 @@ def kickoff_for_each(
         return results
 
     async def kickoff_async(
-        self, inputs: Optional[Dict[str, Any]] = {}
+        self, inputs: Optional[CrewOutput] = {}
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)